__tests__/app.test.js

```diff
@@ -0,0 +1,37 @@
+const request = require('supertest');
+const app = require('../app');
+const testData = require('../db/data/test-data');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+
+afterAll(() => {
+  if (db.end) db.end();
```

Comment:
No need for the conditional here, just do return db.end()

Reply:
Done